src/chttpd/src/chttpd.erl

```diff
@@ -437,32 +525,42 @@ maybe_log(#httpd{} = HttpReq, #httpd_resp{should_log = true} = HttpResp) ->
     % - client port
     % - timers: connection, request, time to first byte, ...
     % - response size
-    % 
-    ?LOG_NOTICE(#{
-        method => Method,
-        path => RawUri,
-        code => Code,
-        user => User,
-        % req_size => MochiReq:get(body_length),
-        src => #{ip4 => Peer},
-        duration => RequestTime
-    }, #{domain => [chttpd_access_log]}),
-    couch_log:notice("~s ~s ~s ~s ~s ~B ~p ~B", [Host, Peer, User,
-        Method, RawUri, Code, Status, RequestTime]);
+    %
+    ?LOG_NOTICE(
+        #{
```

Comment:
+2 to symmetric brackets.

Reply:
+2

Reply:
\cc @awalterschulze I was linking to this in https://github.com/WhatsApp/erlfmt/issues/292#issuecomment-845958607